lib/src/component2_suggestors/component2_constants.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:meta/meta.dart';
+
+const revertInstructions = '/// These updates are opt-in and can be '
+    'rolled back if you do not wish to do the update at this time.';
+
+String methodDeprecationMessage({@required methodName}) {
+  final updateInstructionLink =
+      'https://reactjs.org/docs/react-component.html#updating-$methodName';
```

Comment:
When i try and hit that url it doesn't seem to take me to any section... maybe update it to this?
```suggestion
      'https://reactjs.org/docs/react-component.html#unsafe_${methodName.toLowerCase()}';
```

Reply:
I think they must have switched that anchor off just recently. I pulled the URL from the doc comment in `react.dart` and had tested it multiple times, but now it's not working 🤷‍♂ Good catch!